keps/sig-node/2133-kubelet-credential-providers/kep.yaml

```diff
@@ -22,12 +22,12 @@ replaces:
   - "/keps/sig-cloud-provider/20191004-out-of-tree-credential-providers.md"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: beta
```

Comment:
Diff doesn't show it, but beta graduation was already merged in a previous PR and listed here:

https://github.com/kubernetes/enhancements/blob/9c65705f22ecc31f72f744be21a318c012c1e1f2/keps/sig-node/2133-kubelet-credential-providers/README.md#alpha---beta-graduation